### DISCUSSION TITLE
Welcome to mediasoup-sfu-webrtc-video-rooms Discussions!

### DISCUSSION THREAD

**Discussion by Dirvann** [Dirvann/mediasoup-sfu-webrtc-video-rooms#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rezahabibijan** [comment]
Hello , 
I am improving your repository , and  i implemented recording feature (Video and Audio ) 
I want to have corporation with you
